backend/graminsta/post/services.py

```diff
@@ -0,0 +1,40 @@
+# -*- coding: UTF-8 -*-
+"""services.py"""
+
+from django.contrib.auth import models as auth_models
+from django.shortcuts import get_object_or_404
+from .serializers import UserSerializer
+from .serializers import FollowSerializer
+from .models import FollowRelationship
+
+
+def get_people_user_follows(username):
+    """
+    Returns JSON format users that the given user follows.
+    """
+    user = auth_models.User.objects.get(username=username)
+    ids = FollowRelationship.objects.filter(from_user=user) \
+        .values_list('to_user_id', flat=True)
+    queryset = auth_models.User.objects.filter(id__in=ids)
+    serializer = UserSerializer(queryset, many=True)
+    return serializer
+
+
+def create_follow_relationship(validated_data):
```

Comment:
Services should take parameters explicitly. Do not use `validated_data`, I would suggest to enforce this in other PRs too. @shiyi001 @isabella0428 @hht-hannah 

For example:
```
def follow(request_user, target_user):
  """Creates follow relationship between request_user and target_user.

  Parameters
  ------------
  request_user: The user who triggered this action.
  target_user: The user who will be followed by the request_user.
  ...
```